Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:16.04 as builder
```

Comment:
Can we bump this to 18.04? That's the latest LTS.

Reply:
(also the runtime container below)

Reply:
fixed